cohort_4/playbook/ACTIVITIES.md

```diff
@@ -1431,3 +1432,21 @@ Want to send a link to everyone? _Slack._
 
 Was there an event at Galvanize that had some extra food and offered it to our
 hackers? _Slack._
+
+## Additional Resources
+
+### CodeHS
+
+#### Pros:
+
+- is very good at introducing and meticulously explaining the basic
+concepts of programming I (Jonathan) have not found anything clearer than
+CodeHS at explaining these basics.
+
+#### Cons:
+
+- the student nevers have the opportunity to build your own projects in CodeHS.
+  You are told everything that you are supposed to build.
+- perhaps even worse, you have no experience even in building _any_ kind
+  projects because you are always following a specific directions that keep
+  giving you the next step.
```

Comment:
Perhaps this could be included in a separate pull request? It seems out of
scope of this one.

On Sat, Aug 22, 2015 at 6:40 PM, Selynna Sun <notifications@github.com>
wrote:

> In cohort_4/playbook/ACTIVITIES.md
> <https://github.com/hackedu/hack-camp/pull/519#discussion_r37701723>:
>
> > +
> > +### CodeHS
> > +
> > +#### Pros:
> > +
> > +- is very good at introducing and meticulously explaining the basic
> > +concepts of programming I (Jonathan) have not found anything clearer than
> > +CodeHS at explaining these basics.
> > +
> > +#### Cons:
> > +
> > +- the student nevers have the opportunity to build your own projects in CodeHS.
> > +  You are told everything that you are supposed to build.
> > +- perhaps even worse, you have no experience even in building _any_ kind
> > +  projects because you are always following a specific directions that keep
> > +  giving you the next step.
>
> We used CodeHS for Cohort 2
>
> @jonleung <https://github.com/jonleung>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hack-camp/pull/519/files#r37701723>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us